src/hotspot/share/gc/shared/cardTable.cpp

```diff
@@ -73,7 +73,6 @@ CardTable::CardTable(MemRegion whole_heap) :
   _byte_map_size(0),
   _byte_map(nullptr),
   _byte_map_base(nullptr),
-  _cur_covered_regions(0),
   _covered(MemRegion::create_array(_max_covered_regions, mtGC)),
   _committed(MemRegion::create_array(_max_covered_regions, mtGC)),
```

Comment:
`_covered` and `_committed` are also referenced in `vmStructs_gc.hpp`. I can use `MemRegion _covered[2];` inside `class CardTable`, but how to represent that array-type in `vmStructs_gc.hpp`?

Reply:
They are not used in the hotspot agent afaict though, so they could simply be removed.

Otherwise one could use two separate members of `MemRegion` type.